api/devices/schema.py

```diff
@@ -78,8 +78,6 @@ def mutate(self, info, device_id, **kwargs):
         ).first()
         if not exact_device:
             raise GraphQLError("DeviceId not found")
-        exact_device.date_added = datetime.now()
```

Comment:
@vic3king Great job, however, do you think we should update the `date_added` field?

Reply:
@codinger41 I removed that line. we are no longer updating the `date-added`

Reply:
Oh sorry, great job!